Resources/Changelog/Changelog.yml

```
@@ -12402,3 +12402,83 @@ Entries:
   id: 6911
   time: '2025-03-07T22:45:53.0000000+00:00'
   url: https://github.com/Simple-Station/Einstein-Engines/pull/1885
+- author: Eagle
+  changes:
+    - type: Add
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)